src/org/aavso/tools/vstar/ui/mediator/message/MessageBase.java

```diff
@@ -22,6 +22,8 @@
  */
 public class MessageBase {
 
+	private String name;
```

Comment:
I'm wondering about the use of `name` here. The intent is something like "destination" or "recipient" identifier I think. Is that correct?